src/scripts/main.js

```diff
@@ -1 +1,13 @@
 'use strict';
+
```

Comment:
https://user-images.githubusercontent.com/38065883/174635451-1417a4e4-3f50-45e1-8b00-ee7f3b6ce3a7.mov

Page isn't scrollable from the very beginning, cause for some reason its already has `with-menu` class